x-pack/test/functional/apps/maps/sample_data.js

```diff
@@ -80,15 +89,25 @@ export default function ({ getPageObjects, getService, updateBaselines }) {
       await kibanaServer.uiSettings.update({
         [UI_SETTINGS.TIMEPICKER_QUICK_RANGES]: SAMPLE_DATA_RANGE,
       });
+      await security.testUser.setRoles(
+        ['global_maps_all', 'geoall_data_writer', 'kibana_sample_admin'],
```

Comment:
Do we still need 'geoall_data_writer' here?  It's not clear to me why.

Reply:
yes we do. If I we don't - we can't add the sample data index patterns.

Reply:
But the sample data was already loaded before this step as the default superuser.